docs/content/components/prompts/single-choice.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Single choice
+titleTemplate: ":title · Prompts · Noora · Tuist"
+description: A component to select an option among many.
+---
+
+# Single choice
+
+This component is a simple component that prompts the user to select a single option among many. It is a good way to get a quick answer from the user.
+
+| Property | Value |
+| --- | --- |
+| Interactivity | Required (fails otherwise) |
```

Comment:
This doesn't feel right. Should this be removed?

Reply:
It's right now, so I'd keep it. We want to make sure people don't use these components in non-interactive environments, otherwise they might experience their CLIs hanging.